js/chart.js

```diff
@@ -0,0 +1,220 @@
+// Filter out an array with the users in past 60 minutes
+function filter(data) {
+  //this gives you an array of tuples where the first element is the hour and the last element is the # of users
+  var result = data.rows.map(
+    function(elem) {
+      return [elem[0], elem[1], elem[2]]
+    })
+
+  var date = new Date();
+  var hour = date.getHours();
+  var minute = date.getMinutes();
+  var x = [];
+
+  // sort result by hour and minute in ascending order
+  result.sort(function(a, b) {
+    var n = a[0] - b[0];
+    if(n !== 0) {
+      return n;
+    }
+    return parseInt(a[1]) - parseInt(b[1]);
+  });
+
+  // push number of pageviews in the past 60 minutes to an array
+  var counter = minute;
+  for(var i = 0; i < result.length; i++) {
+    // reset counter after 60 min mark
+    if(counter == 59) {
+      counter = 0;
+    }
+    if((result[i][0] == (hour-1)) && (result[i][1] >= counter)) {
+      x.push(Number(result[i][2]));
+      counter++;
+    }
+  }
+  // filtered array in past hour
+  return x;
+}
+
+var request = $.ajax({
```

Comment:
Don't need to save the ajax object if you are going to define the success event in the ajax definition.